src/library/Uno.Material/Styles/Application/AnimationConstants.xaml

```diff
@@ -7,4 +7,9 @@
 	<CubicEase x:Key="Material_EaseOutFunction"
 			   EasingMode="EaseOut" />
 
+	<Duration x:Key="Material_AnimationDuration">0:0:0.25</Duration>
```

Comment:
I see it was done previously but let's not use underscore (  _   ) in our variable names.